text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala

```diff
@@ -146,8 +146,9 @@ object ExtractAndAlign {
 //    }
 //    pw.close()
     // ----------------------------------
-
-    val commentToTextAlignments = w2vAligner.alignMentions(commentDefinitionMentions, textDefinitionMentions)
+    val defEditAligner = new MentionEditDistanceAligner
```

Comment:
if you can specify this in the config file -- bonus points

Reply:
@maxaalexeeva  Are you planning on going for the bonus points? Or should I merge?

Reply:
Bonus points. Thanks for checking! I'll mention you here when it's ready to merge.